index.js

```diff
@@ -643,7 +644,7 @@ const writeAlerts = async(client, alerts) => {
       }
       let fields =  { message };
       if (target_sid) fields = Object.assign(fields, {target_sid});
-      const obj = {measurement: 'alerts', fields: fields, tags: { alert_type, service_provider_sid, account_sid }};
+      const obj = {measurement: 'alerts', fields: fields, tags: { alert_type, service_provider_sid, account_sid, vendor }};
```

Comment:
Since we will not always have a vendor property and influxdb does not support null values, I would prefer
```js
const obj = {measurement: 'alerts', fields: fields, tags: {
   alert_type, service_provider_sid, account_sid,
   ...(vendor && {vendor}}};
```